integreat_cms/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -147,6 +157,9 @@ <h1 class="heading">{% translate "Technical Feedback" %}</h1>
                     <option data-bulk-action="{% url 'mark_admin_feedback_as_unread' %}">
                         {% translate "Mark as unread" %}
                     </option>
+                    <option data-bulk-action="{% url 'archive_admin_feedback' %}">
+                        {% translate "Archive" %}
+                    </option>
```

Comment:
```suggestion
                    {% if perms.cms.change_feedback %}
                        <option data-bulk-action="{% url 'archive_admin_feedback' %}">
                            {% translate "Archive" %}
                        </option>
                    {% endif %}
```

Marketing team users can view but cannot change feedback. I think we can hide the archive bulk action to them.

Reply:
Ah, I guess then we can hide the whole list of bulk actions, because they don't have access to any actions